src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertificateOrderConfirmationMapper.java

```diff
@@ -41,14 +41,25 @@
     @Mapping(source = "deliveryDetails.region", target="city")
     @Mapping(source = "deliveryDetails.postalCode", target="postCode")
     @Mapping(source = "deliveryDetails.country", target="country")
-
-    // Order details field mappings
     @Mapping(source = "reference", target="orderReferenceNumber")
     @Mapping(source = "orderedBy.email", target="emailAddress")
     @Mapping(source = "totalOrderCost", target="feeAmount")
-
     public abstract CertificateOrderConfirmation orderToConfirmation(OrderData order);
```

Comment:
MapStruct annotations can be removed if the method is implemented as follows:

`    
public CertificateOrderConfirmation orderToConfirmation(OrderData order) {
        return orderToConfirmation(order, null);
    }
`